### ISSUE TITLE
1 Question

### ISSUE THREAD

**Issue by OldGamer87** [1oom-fork/1oom#79]
I understand one things in MoO.
I played on hard, early games i meet alien races (honorable, pacifist) and we have good relations ( diplomacy bar "relaxed", trade, no spying).
AND Surprise! Enemy races send transport to my new colonies. Killing population and capture planet. Relations bar is green ok (relaxed) Why? 
And I send transport, recapture my planet, and enemy emperor is angry and declared war. It does not make sense (?)
This occurs also old MSdos version.  Even pacifist psilon or honorable Alkari.. This is one thing which not understand, and a little he's spoiling my fun. Someone knows the reason??

**Comment by 1oom-fork** [comment]
Everything is very simple here. AI does what it sees fit. These are the rules in MOO1

**Comment by 1oom-fork** [comment]
Seriously though, I don't know if this is intended or there is a mistake somewhere

**Comment by 1oom-fork** [comment]
In any case, if you are upset that the AI ​​is breaking the rules, you are better off choosing another game. Sometimes this game gets completely crazy and it's great.

**Comment by OldGamer87** [comment]
No, no ,MoO is too much great games in change to another space strategy games. 
I always will be love them. I'm looking forward to every update. 
Currently exists only one games which like, its "Stars in Shadow". Its good game, but not is MoO level :D
Rest title space strategy is hybrid turn-RTS crap in my opinion.
Regards! :D

**Comment by 1oom-fork** [comment]
It’s a mystery to me why most authors cut out air combat from modern games... That’s the whole point

**Comment by OldGamer87** [comment]
Yes.. I think that turn battle in game must be fun, plaesant, curious. Taking example Heroes 1-3, or MoO2. This games have fantastic battle, with simple 2d graphic without zoom\rotate cameras.  MoO2 have I'think best combat system, with many possibilities. New MoO 2016 have tragic battles, very boring. Only graphic is nice (races, ship models). I don't know if anyone still plays it.

**Comment by OldGamer87** [comment]
I have actually short holodays (1 week) and extremaly intensively playing MoO days and night :). I remembered why leave this game few years ago.
Exactly this problem wchich described above. This game of work of genius. But there is a chance to fix AI behavior with regard
sends transport to player colonies in early games? AI captured planets without consequences, or attack players colonies(even treaty broken). Relations bar remains good/green.
You know problem probably very good. I'm no skill to repair this, I don't know how much work with this. My question : its any
chanse change that in future?Can we hope or is there no chance of repair?
 What do you think? Its very important for me. If it's not the people from the 1oom fork, no one will do it anymore.

**Comment by 1oom-fork** [comment]
I think you should look at the problem from a different perspective. I am increasingly convinced that the original rules are better than the modern ones, with all their shortcomings. Anything can be fixed, maybe I’ll do it, but it’s not worth it. If someone points out where exactly the error lies, I'm ready to fix it as soon as I check everything.

**Comment by 1oom-fork** [comment]
I would also like to add that at this stage of development, fixing original errors is a minimum priority for me, I correct them only if I clearly and fully understand the problem, on the contrary, I strive as a priority to restore all errors corrected in 1oom from the original.

**Comment by 1oom-fork** [comment]
Ideally, it would be nice if you fixed everything yourself. I would be happy

**Comment by 1oom-fork** [comment]
And don’t tell me stories about how you’re not capable of this.

**Comment by 1oom-fork** [comment]
And one more thing. Thank you in advance for your efforts, I am confident in your success. :)
